src/swell/configuration/jedi/interfaces/geos_marine/model/getvalues.py

```diff
@@ -0,0 +1,19 @@
+# (C) Copyright 2021- United States Government as represented by the Administrator of the
+# National Aeronautics and Space Administration. All Rights Reserved.
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+# --------------------------------------------------------------------------------------------------
+
+from collections.abc import Mapping
+
+# --------------------------------------------------------------------------------------------------
+
+
+def getvalues(template_dict: Mapping) -> Mapping:
+    getvalues = None
+
+    return getvalues
```

Comment:
This should fail type checking (`None` is not a `Mapping`). If we want to be consistent with these always returning `Mapping`s (which is a good idea), then:


```suggestion
def getvalues(_: Mapping) -> Mapping:
    return {}
```

(Also note: Python convention is to use an `_` for an unused variable. Without this, most linters should raise "unused variable" warnings).

Reply:
I added `Optional` to the type hint - in this case the file does need to return None